channel/mod.go

```diff
@@ -0,0 +1,58 @@
+// Package channel defines the logger.
+//
+// channel is using a global logger with some default parameters.
+// It is disabled by default and the level can be increased using
+// an environment variable:
+//
+//	CRY_LOG=trace
+//	CRY_LOG=info
+package channel
+
+import (
+	"os"
+	"time"
+
+	"github.com/rs/zerolog"
+)
+
+// EnvLogLevel is the name of the environment variable to change the logging
+// level.
+const EnvLogLevel = "CRY_LOG"
```

Comment:
do we want a separate configuration for `channel` and the `sync` package?
I'm thinking that they could work hand in hand, and we could have a single environment flag.

Plus, if needed, I would make it so that the log level can be set programmatically (if we really need more fine-tuning).

Reply:
Why not but how does it work with different package names (sync and channel) ? Doesn't it require to have the same package name (debugtools) ?

Reply:
I'm not sure I understand the question 🤔